sources/packages/forms/src/form-definitions/educationprogram.json

```diff
@@ -1580,7 +1580,7 @@
                     "allowCalculateOverride": false,
                     "validate": {
                         "required": true,
-                        "onlyAvailableItems": false,
+                        "onlyAvailableItems": true,
```

Comment:
We need to review if this option really works for `"type": "selectboxes"`. I can see the option available in the form.io [Sandbox](https://formio.github.io/formio.js/app/sandbox.html) but it is causing issues in the UI right now.

![image](https://github.com/bcgov/SIMS/assets/61259237/a17d166f-b02c-41ed-ada6-7617a04165b8)


![image](https://github.com/bcgov/SIMS/assets/61259237/9399f976-c888-4cc9-b197-49c2e012c817)

Reply:
The `onlyAvailableItems` will not work for `selectboxes`  and it will cause a validation issue if set to `true`.
Apparently `onlyAvailableItems` was introduced targeting only `select` and `radio` components.
When `selectboxes` are used they will generate a submission like the below one.
```json
 "programDeliveryTypes": {
  "deliveredOnSite": true,
  "deliveredOnline": true
},
```
 If the below is tempered by a malicious user form.io validation server will return a server error.
```json
{
  "message": "Unexpected token y in JSON at position 284",
  "error": "Bad Request",
  "statusCode": 400
}
```
Updating the AC of the ticket to remove the `"type": "selectboxes"`.

Reply:
all selectboxes should now have onlyAvailableItems set to false